libs/SalesforceSDKCore/SalesforceSDKCore/Classes/IDP/SFSDKLoginFlowSelectionViewController.m

```diff
@@ -223,7 +223,7 @@ - (void)styleNavigationBar:(UINavigationBar *)navigationBar {
     if (!navigationBar) {
         return;
     }
-    [SFSDKViewUtils styleNavigationBar:navigationBar config:[SFUserAccountManager sharedInstance].loginViewControllerConfig];
+    [SFSDKViewUtils styleNavigationBar:navigationBar config:[SFUserAccountManager sharedInstance].loginViewControllerConfig classes:@[[self.navigationController class]]];
```

Comment:
Passing in the scoping class based on [this thread](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/3449#discussion_r759752770). I wasn't able to scope based on the view controller because the nav bar is higher than it in the hierarchy so I used the nav controller which ends up being SFSDKNavigationController, if there's a better way let me know

Reply:
I've validated these changes against FSL - looks good @bbirman.